composer.json

```
@@ -11,6 +11,12 @@
       "Pantheon\\EI\\": "src"
     }
   },
+  "scripts": {
+    "test": "phpunit tests"
```

This uses whatever `phpunit` is installed on the machine. We shouldn't assume that phpunit is installed outside the repository. While running the `composer install` first, as mentioned in the README, seems to get around this, we can be explicit about which phpunit we're using:

```suggestion
    "test": "vendor/bin/phpunit tests"
```

Adding a `phpunit.xml` can eliminate the need to pass the directory into the command, too (really simple example config file [here](https://github.com/pantheon-systems/pantheon-wordpress-edge-integrations/blob/fe2b906993d668812e09bc6d45ce8e4d8dd7d725/phpunit.xml)), in which case the command could be:

```json
"scripts": {
    "test": "vendor/bin/phpunit -c phpunit.xml"
},
```

In the future, we may add some more/other tests. In that case, we'd probably do something like:

```json
"scripts": {
    "test:phpunit": "vendor/bin/phpunit -c phpunit.xml",
    "test:testcafe": "npm run testcafe . ",
    "test": "composer test:phpunit && composer test:testcafe",
},
```

...but mapping `test` to just the phpunit tests is probably fine until then. If this library stays 100% PHP, there will likely not be a need for anything else.

---

Great suggestions, Chris!